programs/drift/src/controller/orders.rs

```diff
@@ -1803,8 +1806,13 @@ pub fn fulfill_perp_order_with_amm(
     let user_position_delta =
         get_position_delta_for_fill(base_asset_amount, quote_asset_amount, order_direction)?;
 
+    crate::dlog!(split_with_lps);
     if split_with_lps {
         update_lp_market_position(market, &user_position_delta, fee_to_market_for_lp.cast()?)?;
+    } else {
+        msg!("HIHIHI {:?} {:?}", fee_to_market, fee_to_market_for_lp);
+        fee_to_market = fee_to_market.safe_add(fee_to_market_for_lp)?;
```

Comment:
does this mean the protocol was making less than it should have been in fee?

Reply:
i suspect that, just the total_fee_minus_distribution might be off if there were LPs (since it assumed it was split with LPs?)

Reply:
reverted was mistaken